[WIP] Draft charter (#2)
DevRel-Foundation/wg-innovation-emerging-trends: README.md

Reviewer: One observation - I anticipate that y'all will work hand in hand with other groups (career, resource aggregation, structure & Org), as your output will most likely inform their priorities and updates. I'd love inclusion of how you may achieve this cross-group influence and alignment!

This group aims to:

1. **Explore AI in DevRel**: Investigate practical applications of AI in automation, development, onboarding, and community, enhancing both efficiency and developer satisfaction.
1. **Maintain Relevance**: Examine evolving industry trends to ensure DevRel strategies remain aligned with developers’ needs and expectations.
1. **Balance Sustainability and Reach**: Analyze the trade-offs between in-person and virtual engagement to propose sustainable practices without compromising impact or inclusivity.
1. **Evaluate Engagement Formats**: Compare the effectiveness of live versus asynchronous and virtual versus in-person interactions to inform best practices for developer engagement.

The group will operate transparently, leveraging its mailing list, Discord channel, and GitHub to encourage open participation, knowledge sharing, and collaboration among all stakeholders. Through this work, the group will contribute to the broader mission of the Developer Relations Foundation. 

## Expectations

<!-- Estimate what resources and personnel are needed to accomplish the mission statement -->

> Members should try to allocate an hour per week to working group activities. This may include:

Reviewer 2: ```suggestion
Working Group participants should try to allocate an hour per week to working group activities. This may include:
```
Changing members to participants. Under LF ecosystem, the word members can cause confusion, as it typically refers to organizations that sponsor the foundation or project. To date, DRF is a community project with no membership /sponsorship model.

|    ---   |     ---   |      ---        |     ---    |
| Shiny Zhu | [@shinyzhu](https://github.com/shinyzhu) | shinyzhu | [🔗](https://www.linkedin.com/in/shinyzhu/) |
| Benjamin Odom | [@bjodom](https://github.com/bjodom) | benodom | [🔗](https://www.linkedin.com/in/benjamin-odom-07539180/) |
| David Simmons | [@davidgs](https://github.com/davidgs) | davidgs | [🔗](https://www.linkedin.com/in/davidgsimmons/) |
| Luke Oliff | [@lukeocodes](https://github.com/lukeocodes) | lukeocodes | [🔗](https://www.linkedin.com/in/lukeocodes/) |

If you have any questions, please check the [DRF FAQ](https://github.com/DevRel-Foundation#faq) first. **If your question is not answered in the FAQ**:

- For questions specific to this working group, contact managers (at) dev-rel.org or use {insert other contact methods}
- For broader topics, reach out to steering-committee (at) dev-rel.org

## Meetings

Our regular will alternate between APAC, EMEA, and AMER, occuring every two weeks. You can find each agenda on GitHub and event in the global DRF community calendar: https://lists.dev-rel.org/g/community/calendar

## Code of Conduct

<!-- inherit DRF COC -->

> Members and participants must adhere to the [code of conduct](https://github.com/DevRel-Foundation/governance/blob/main/code_of_conduct.md) for the wider community.

Reviewer 2: ```suggestion
Working group managers and participants must adhere to the [code of conduct](https://github.com/DevRel-Foundation/governance/blob/main/code_of_conduct.md) for the wider community.
```

> 

Reviewer 2: ```suggestion
## Governance

All Working Groups within the DevRel Foundation must:

- Function with openness and transparency, meaning that participation is open to all, and minutes and other documents are available and easily accessible to everyone.

- Adhere to the DevRel Foundation’s [Code of Conduct](https://github.com/DevRel-Foundation/governance/blob/main/code_of_conduct.md)

- Follow Linux Foundation's [Antitrust Policy](https://www.linuxfoundation.org/legal/antitrust-policy)

- Follow [Chatham House Rule](https://www.chathamhouse.org/about-us/chatham-house-rule) and as otherwise set up in the [Developer Relations Foundation Charter](https://github.com/DevRel-Foundation/governance/blob/main/Technical_Charter_v1.0.adoc)

For More information: <https://github.com/DevRel-Foundation/governance/pull/17/files#diff-a5e748f515fc060f9baf9feb8e2871c97cd5657858b78f583c6c025c7e49f176>

Compliance with Policies This Charter is subject to the Series Agreement for the Project and the Operating Agreement of LF Projects. Contributors will comply with the policies of LF Projects as may be adopted and amended by LF Projects, including, without limitation the policies listed at <https://lfprojects.org/policies/>.
```

Welcome to the DRF Innovation and Emerging Trends Work Group

> - Our mailing list to join is here: https://lists.dev-rel.org/g/wg-innovation-emerging-trends

Reviewer 3: ```suggestion
- Our mailing list to join is [here](https://lists.dev-rel.org/g/wg-innovation-emerging-trends).
```
